parsers/examples/example_parser.py

```diff
@@ -300,43 +314,44 @@ def fetch_exchange(
 
-    r = session or Session()
-    url = "https://api.someservice.com/v1/exchange/latest?" "from={}&to={}".format(
-        zone_key1, zone_key2
-    )
+    url = f"https://api.someservice.com/v1/exchange/latest"
+    params = {
+        "from": zone_key1,
+        "to": zone_key2,
+    }
 
-    response = r.get(url)
-    assert response.status_code == 200
+    response = session.get(url, params=params)
+    assert (
+        response.status_code == 200
+    ), f"Exception when fetching exchange for {zone_key1} -> {zone_key2}: error when calling url={url}"
     obj = response.json()
 
-    data = {
-        "sortedZoneKeys": "->".join(sorted([zone_key1, zone_key2])),
-        "source": "someservice.com",
-    }
-
-    # Zone keys are sorted in order to enable easier indexing in the database
-    sorted_zone_keys = sorted([zone_key1, zone_key2])
-    # Here we assume that the net flow returned by the api is the flow from
-    # country1 to country2. A positive flow indicates an export from country1
-    # to country2. A negative flow indicates an import.
-    net_flow = obj["exchange"]
-    # The net flow to be reported should be from the first country to the
-    # second (sorted alphabetically). This is NOT necessarily the same
-    # direction as the flow from country1 to country2
-    data["netFlow"] = net_flow if zone_key1 == sorted_zone_keys[0] else -1 * net_flow
+    exchange_list: List[dict] = []
 
-    # Parse the datetime and return a python datetime object
-    data["datetime"] = arrow.get(obj["datetime"]).datetime
+    for item in obj:
+        exchange_list.append(
+            {
+                # Zone keys are sorted in order to enable easier indexing in the database
+                "sortedZoneKeys": "->".join(sorted([zone_key1, zone_key2])),
+                # Parse the datetime and return a python datetime object
+                "datetime": datetime.fromisoformat(item["datetime"]),
+                # Here we assume that the net flow returned by the api is the flow from
+                # country1 to country2. A positive flow indicates an export from country1
+                # to country2. A negative flow indicates an import.
+                "netFlow": item["exchange"],
+                "source": "someservice.com",
+            }
+        )
 
-    return data
+    return exchange_list
 
 
 if __name__ == "__main__":
     """Main method, never used by the Electricity Map backend, but handy for testing."""
 
-    print("fetch_production() ->")
-    print(fetch_production())
-    print("fetch_price() ->")
-    print(fetch_price())
-    print("fetch_exchange(DK, NO) ->")
-    print(fetch_exchange("DK", "NO"))
+    print("fetch_production(DK-DK1) ->")
+    print(fetch_production("DK-DK1"))
+    print("fetch_price(DK-DK1) ->")
+    print(fetch_price("DK-DK1"))
+    print("fetch_exchange(DK-DK1, DK-DK2) ->")
+    print(fetch_exchange("DK-DK1", "DK-DK2"))
```

Comment:
Should we continue with the non-specific values here?

```suggestion
    print("fetch_production(XX) ->")
    print(fetch_production("XX"))
    print("fetch_price(XX) ->")
    print(fetch_price("XX"))
    print("fetch_exchange(XX, YY) ->")
    print(fetch_exchange("XX", "YY"))
```

Reply:
I wanted something a little more abstract and general but it might be better to go back to the DK zones as examples as there is really no substitute for the time zones anyway.